config/scout_apm.exs

```diff
@@ -0,0 +1,12 @@
+# This configuration file is used for Scout APM.
+# See our help docs at http://help.apm.scoutapp.com for more information.
+# config/scout_apm.exs
+use Mix.Config
+
+config :scout_apm,
+  name: System.get_env("SCOUT_APP_NAME") || "", # The app name that will appear within the Scout UI
+  key: System.get_env("SCOUT_APP_KEY") || ""
```

Comment:
You should maybe add these to `.env.example`

Reply:
Added.